docs/developers/tutorial.md

```diff
@@ -640,16 +641,77 @@ The expansion rules for `github:org/repo` are the following:
 
 ## Publish the rule to Maven Central
 
-The most robust way to share a custom rule is to publish it as a library to
-Maven Central. The
-[`build.sbt`](https://github.com/olafurpg/named-literal-arguments/blob/master/scalafix/build.sbt)
-shows the necessary changes to build.sbt to populate publishing information. The
-[sbt-ci-release](https://github.com/olafurpg/sbt-ci-release) readme documents
-the further steps to configure gpg and Sonatype.
+Run the following sbt command to publish a rule locally.
 
-Once you have your rule, users can depend on it in the sbt plugin by updating
-`scalafixDependencies`
-(`scalafix.sbt.ScalafixPlugin.autoImport.scalafixDependencies`)
+```sh
+> rules/publishLocal
```

Comment:
cc/ @joan38 clarified how to publish rules

Reply:
Thanks!